Packs/Tidy/Integrations/Tidy/Tidy.py

```diff
@@ -657,8 +656,9 @@ def main() -> None:
 
     # Tidy client configuration
     hostname = demisto.getArg("hostname") or demisto.getParam("hostname")
-    user = demisto.getArg("user") or demisto.getParam("user")
-    password = demisto.getArg("password") or demisto.getParam("password")
+    user = demisto.getArg("user") or demisto.params().get('user_creds', {}).get('identifier') or demisto.params().get("user")
```

Comment:
hidden args cant take precedence

Reply:
thats how it was originally, should i change?

Reply:
didnt realize it was arg then param then old param. lets keep